db/migrate/20221025191034_create_users.rb

```diff
@@ -1,9 +1,9 @@
-class CreateUser < ActiveRecord::Migration[7.0]
+class CreateUsers < ActiveRecord::Migration[7.0]
   def change
     create_table :users do |t|
       t.string :name
-      t.string :photo
-      t.string :bio
+      t.text :photo
+      t.text :bio
       t.integer :posts_counter
```

Comment:
Please correct this line like this to remove the bug

` t.integer :posts_counter, default: 0`